examples/datasource_mapping/resources.yaml

```diff
@@ -0,0 +1,241 @@
+---
+apiVersion: grafana.integreatly.org/v1beta1
+kind: Grafana
+metadata:
+  name: grafana
+  labels:
+    dashboards: "grafana"
+spec:
+  client:
+    preferIngress: true
+  config:
+    log:
+      mode: "console"
+    auth:
+      disable_login_form: "false"
+    security:
+      admin_user: root
+      admin_password: secret
+---
+apiVersion: grafana.integreatly.org/v1beta1
+kind: GrafanaDatasource
+metadata:
+  name: grafanadatasource-sample1
+spec:
+  instanceSelector:
+    matchLabels:
+      dashboards: "grafana"
+  datasource:
+    name: prom1
+    type: prometheus
+    access: proxy
+    url: http://prometheus-service:9090
+    isDefault: true
+    jsonData:
+      'tlsSkipVerify': true
+      'timeInterval': "5s"
+    editable: true
+---
+apiVersion: grafana.integreatly.org/v1beta1
```

Comment:
Why do we need the first sampledatasource?
Currently it only confuses me.

Reply:
The intention was to demonstrate that you can select a non-default datasource with this mapping. If there was only one data source, I believe it would be picked by default.

Reply:
I wrote a small note about it in the README.